client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -19,7 +21,17 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 			'You enabled Jetpack and unlocked dozens of website-bolstering features. Continue preparing your site below.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		const siteRedirectHref = '/jetpack/onboarding/site-title';
```

Comment:
Let's add a TODO comment here, to remember to fix this one. Maybe to avoid confusion, just make it a `#` for the time being.